tests/docs/smoke-all/conditional-content/meta.qmd

```diff
@@ -0,0 +1,71 @@
+---
+title: "conditional content on meta"
+check_true: true
+check_false: false
```

Comment:
Are these properties visible to content incorporated using the `include` short code?

How are properties in the parent YAML combined with properties in the YAML of an included file?

Reply:
Perfect. Thanks. I wasn't sure about the order-of-operations.